src/install/bin.zig

```diff
@@ -319,14 +320,22 @@ pub const Bin = extern struct {
             var remain: []u8 = &dest_buf;
 
             if (!link_global) {
-                target_buf[0..".bin/".len].* = ".bin/".*;
-                from_remain = target_buf[".bin/".len..];
-                dest_buf[0.."..".len].* = "..".*;
-                remain = dest_buf["..".len..];
-                std.mem.copy(u8, remain, this.package_installed_path);
-                remain = remain[this.package_installed_path.len..];
+                const root_dir = std.fs.Dir{ .fd = this.root_node_modules_folder };
+                const from = root_dir.realpath(dot_bin, &target_buf) catch |err| {
```

Comment:
this realpath call feels suspicious

could we use bun.getFdPath instead? Note that `open` will already follow paths/symlinks

Reply:
I want `root/path/.bin` rather than `root/path`, which `bun.getFdPath()` cannot provide directly.

Would you rather prefer `bun.getFdPath(root_dir.openDir(".bin").fd, &target_buf)` here? :raised_eyebrow: